src/components/MovieCard/MovieCard.tsx

```diff
@@ -2,6 +2,51 @@ import React from 'react';
 
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
+import { Movie } from '../../types/Movie';
+
+type Props = {
+  movie: Movie;
+};
+
+export const MovieCard: React.FC<Props> = ({ movie }) => (
+  <div className="card" data-cy="Movie">
+    <div className="card-image">
+      <figure className="image is-4by3">
+        <img
+          data-cy="MovieImage"
+          src={movie.imgUrl}
```

Comment:
I think, you can use  destructuring same like we did in previous works, in order to get accessto values of props object
and only write `imgUrl` instead of `movie.imgUrl`